e2e/cypress/integration/enterprise/guest_accounts/system_console_manage_guest_spec.js

```diff
@@ -0,0 +1,172 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. #. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+/**
+ * Note: This test requires Enterprise license to be uploaded
+ */
+import * as TIMEOUTS from '../../../fixtures/timeouts';
+import {getRandomInt} from '../../../utils';
+import users from '../../../fixtures/users.json';
+
+let guest;
+
+function verifyGuest(userStatus = 'Guest ') {
+    // * Verify if Guest User is displayed
+    cy.findAllByTestId('userListRow').should('have.length', 1);
+    cy.findByTestId('userListRow').find('.MenuWrapper a').should('be.visible').and('have.text', userStatus);
+}
+
+describe('Guest Account - Verify Manage Guest Users', () => {
+    before(() => {
+        // # Enable Guest Account Settings
+        cy.apiUpdateConfig({
+            GuestAccountsSettings: {
+                Enable: true,
+            },
+        });
+
+        // # Login as new user
+        cy.loginAsNewUser().then((userResponse) => {
+            guest = userResponse;
+
+            // # Demote the current member to a guest user
+            cy.demoteUser(guest.id);
+        });
+
+        // # Login as SysAdmin
+        cy.apiLogin('sysadmin');
+
+        // # Visit System Console Users page
+        cy.visit('/admin_console/user_management/users');
+    });
+
+    beforeEach(() => {
+        // # Reload current page before each test
+        cy.reload();
+
+        // # Search for Guest User by username
+        cy.get('#searchUsers').should('be.visible').type(guest.username);
+    });
+
+    it('MM-18048 Verify the manage options displayed for Guest User', () => {
+        // * Verify Guest user
+        verifyGuest();
+
+        // # Click on the Manage User option
+        cy.findByTestId('userListRow').find('.MenuWrapper a').should('be.visible').click();
```

Comment:
I think cypress is clicking the link here too fast, and adding a delay of 500ms right before it verifies the test as expected.

Reply:
Thanks for checking. I did not see this issue, but anyways I have added an extra wait before clicking on the Menu option.